validator/client/validator_propose.go

```diff
@@ -107,14 +105,7 @@ func (v *validator) ProposeBlock(ctx context.Context, slot uint64) {
 	}
 
 	// 3. Compute state root transition from parent block to the new block.
-	resp, err := v.proposerClient.ComputeStateRoot(ctx, block)
```

Comment:
Can you add all of this logic back?

Reply:
+1

Reply:
Looks like we merged without adding this back. I'll revert those changes